src/utils.js

```diff
@@ -1,23 +1,24 @@
 import moment from 'moment';
 
 export function isSameDay(currentMessage = {}, diffMessage = {}) {
-  let diff = 0;
-  if (diffMessage.createdAt && currentMessage.createdAt) {
-    diff = Math.abs(moment(diffMessage.createdAt).startOf('day').diff(moment(currentMessage.createdAt).startOf('day'), 'days'));
-  } else {
-    diff = 1;
-  }
-  if (diff === 0) {
-    return true;
+
+  let currentCreatedAt = moment(currentMessage.createdAt);
+  let diffCreatedAt = moment(diffMessage.createdAt);
+
+  if (!currentCreatedAt.isValid() || !diffCreatedAt.isValid()) {
+    return false;
   }
-  return false;
+
+  return currentCreatedAt.startOf('day').isSame(diffCreatedAt.startOf('day'));
```

Comment:
According with [`isSame` documentation](http://momentjs.com/docs/#/query/is-same/), you can limit the granularity directly:

```js
return currentCreatedAt.isSame(diffCreatedAt, 'day')
```